packages/tracing/src/hubextensions.ts

```diff
@@ -186,13 +186,14 @@ export function startIdleTransaction(
   transactionContext: TransactionContext,
   idleTimeout: number,
   finalTimeout: number,
+  heartbeatInterval: number,
```

Comment:
Can we make `heartbeatInterval` an optional parameter? `startIdleTransaction` is publicly exposed and if `heartbeatInterval` is a mandatory param, this would be a breaking change.